sigpro/sigpro_pipeline_demo.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
the demo notebook should be separated from the package. I suggest creating a folder under root `tutorials` and move the notebook there.

Reply:
Got it - notebook has been moved to the tutorials folder.